internal/backend/logger/logger.go

```diff
@@ -37,7 +37,14 @@ func (onPanicHook) OnWrite(ce *zapcore.CheckedEntry, fs []zapcore.Field) {
 }
 
 func New(cfg *Config) (*zap.Logger, error) {
-	cfg.Zap.Level.SetLevel(zapcore.Level(cfg.Level)) // Default = 0 = Info.
+	// Optional override for the default level (info).
```

Comment:
Where is the default level defined? By zap?

Reply:
Yes. Zap uses numbers (https://pkg.go.dev/go.uber.org/zapcore#InfoLevel), and if the field isn't set then the default value is 0, i.e. InfoLevel.